data/json/monsters/marloss.json

```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "id": "mon_marloss_zealot_f",
+    "//": "a human who had some contact with the fungaloid mind, but didnt come out quite right.",
+    "type": "MONSTER",
+    "name": "marloss zealot",
+    "description": "Her eyes lie vacant and spittle foams in her mouth, as she recites from the hymns in rapturous ecstasy.",
```

Comment:
Perhaps you could add something that suggests they `did not come out quite right` from their meeting with the fungaloid mind? Right now the description doesn't really suggest it.

Reply:
I rather leave the Zealot's mental state for the players interpretation. I'll remove the comment instead